gneiss/_formula.py

```diff
@@ -0,0 +1,90 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import pandas as pd
+import statsmodels.formula.api as smf
+from skbio.stats.composition import ilr
+from gneiss.util import match, match_tips, rename_internal_nodes
+from gneiss._summary import RegressionResults
+from gneiss.balances import balance_basis
+
+
+def _process(table, metadata, tree):
+    """ Matches tips, features and samples between the table, metadata
+    and tree.  This module returns the features and samples that are
+    contained in all 3 objects.
+    """
+    _table, _metadata = match(table, metadata)
```

Comment:
Perhaps adding a comment about this will make sense to avoid futures questions about it. BTW I think it makes sense to do this and if you really wanted to overwrite it you could:
`table, metadata, tree = _intersect_of_table_metadata_tree(table, metadata, tree)`